my-app/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_UNSPLASH_KEY=6ee32035bd5e430b923efffa01bdfc51
```

Comment:
Not in a million years. **DO NOT** push `.env` files to your repository.

Reply:
I used gitignore  to ignore it, but fetching didn't work so i had to add it
I merged it by accident  :cold_sweat: